src/libs/Navigation/AppNavigator/createResponsiveStackNavigator/ThreePaneView.js

```diff
@@ -66,12 +66,24 @@ function ThreePaneView(props) {
                                     StyleUtils.displayIfTrue(props.state.index === i),
                                 ]}
                             >
-                                <PressableWithoutFeedback
-                                    style={[styles.flex1]}
-                                    onPress={() => props.navigation.goBack()}
-                                    accessibilityLabel={translate('common.close')}
-                                    accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
-                                />
+                                <View style={[styles.flex1, styles.flexColumn]}>
+                                    {/* The PressableWithoutFeedback with 30px height was added to make whole dimmed area pressable
+                                    on all web devices, but aslo make the top of the desktop apps draggable.
+                                    Please remember that in latestest Electron update, buttons can't be both draggable and clickable  */}
+                                    <PressableWithoutFeedback
+                                        style={[styles.w100, {height: 30}]}
+                                        onPress={() => props.navigation.goBack()}
+                                        accessibilityLabel={translate('common.close')}
+                                        accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
+                                    />
+                                    <PressableWithoutFeedback
+                                        style={[styles.flex1]}
+                                        onPress={() => props.navigation.goBack()}
+                                        accessibilityLabel={translate('common.close')}
+                                        accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
+                                        noDragArea
+                                    />
+                                </View>
```

Comment:
@roryabraham are we good with this?

Reply:
If not we can go with just removing `noDragArea` and have a draggable area like presented [here](https://github.com/Expensify/App/pull/24098#issuecomment-1680598262)